pkg/kn/commands/service/configuration_edit_flags.go

```diff
@@ -102,6 +105,11 @@ func (p *ConfigurationEditFlags) Apply(service *servingv1alpha1.Service, cmd *co
 		return err
 	}
 
+	err = servinglib.UpdateContainerPort(template, p.Port)
```

Comment:
Say I have configured my service to run on port 9090, now I want to update my service with an environment variable, this `Apply` gets called, irrespective of user intent to update the port, this will update the container port to default one (8080).

invoke this operation by flag check 
`if cmd.Flag().Changed('port')` (as done for `image` above)

Reply:
Updated